tasks.py

```diff
@@ -32,10 +32,12 @@
 default_network = "kind"
 extra_network = "network2"
 controller_gen_version = "v0.14.0"
+kubectl_version = "v1.27.0"
+kustomize_version = "v5.4.0"
```

Comment:
We should verify that this is actually the version needed, I could not find which exact version metallb uses

Reply:
I guess the latest should work. What's important is pinning so we'll have reproducible behaviour.

Reply:
sounds good, is there some specific documentation page i should update?